lib/presentation/features/products/bloc_list_data.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:openflutterecommerce/data/abstract/model/category.dart';
+import 'package:openflutterecommerce/data/abstract/model/hashtag.dart';
+import 'package:openflutterecommerce/data/abstract/model/product.dart';
+
+class ProductListData {
```

Comment:
May be we should include here filter options too: price range, available colors, sizes and brands?

Reply:
I mean here that this data is shown on screen somehow. Filters and sort rules are just rules and are not shown in this exact screen. But in fact, it is not necessary to wrap it inside this.